Makefile

```diff
@@ -340,7 +340,7 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-containerProtoVer=v0.2
```

Comment:
so we are not creating our own container?

Reply:
We could create our own container but I think it makes sense to use the tendermint container for now.